fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -129,4 +135,17 @@ public void configure() throws Exception {
                 "{{serialization.binaries}}${headers[CamelSerializationPath]}")
             .to("exec:rm");
     }
+
+    private List<Predicate> getUriFilter() {
```

Comment:
This same `getUriFilter()` code appears to be repeated in four places - would it be reasonable to make it a static method on  ___, e.g. like `getUriFilter(CamelContext cxt)`?